docs/examples/controls/OrbitControls.html

```diff
@@ -56,8 +56,8 @@ <h3>[name]( [param:Camera object], [param:HTMLDOMElement domElement] )</h3>
 		<p>
 			[page:Camera object]: (required) The camera to be controlled.<br><br>
 
-			[page:HTMLDOMElement domElement]: (optional) The HTML element used for event listeners. By default this is the whole document,
-			however if you only want the controls to work over a specific element (e.g. the canvas) you can specify that here.
+			[page:HTMLDOMElement domElement]: The HTML element used for event listeners. In most cases, you want to pass in
+			the renderer's canvas element via *renderer.domElement*.
```

Comment:
>In most cases, you want to pass in...

I do not think that is appropriate in documentation. If you want to go that route, you will have to say which cases.

Reply:
I've removed this sentence.